components/x-gift-article/src/ShareArticleDialog.scss

```diff
@@ -179,3 +187,41 @@
 		@include oTypographySans($weight: 'medium');
 	}
 }
+
+// Stopped using o-overlay but preserved its close functionality
+// Styles copied from o-overlay
+.share-article-modal__close {
+	display: inline-block;
+	background-repeat: no-repeat;
+	background-size: contain;
+	background-position: 50%;
+	background-color: transparent;
+	vertical-align: baseline;
+	width: 20px;
+	height: 20px;
+	background-image: url(https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1:cross?format=svg&source=next-article&tint=%2366605C%2C%2366605C);
+	-webkit-appearance: none;
```

Comment:
**Change request:** I believe our build step should add prefixes based on our browser support so hopefully we don't need to manually add them.

https://github.com/Financial-Times/dotcom-page-kit/blob/main/packages/dotcom-build-sass/src/index.ts#L65


It would be worth checking that this works as expected though, don't just take my word for it